### PR TITLE
docs(NODE-5961): aggregate projection example 1

### PR DESCRIPTION
**Pull request by aditi-khare-mongoDB**
### Description
Add example test case for Aggregate Projection to docs. 

#### What is changing?
Example being added to the documentation for aggregate projections. 

##### Is there new documentation needed for these changes?
Yes, this example will be added under the Node.js tab on the [Project Fields to Return from Query Results](https://www.mongodb.com/docs/manual/tutorial/project-fields-from-query-results/) under Aggregate Projection

#### What is the motivation for this change?
Documentation team request.

### Double check the following

- [x] Ran `npm run check:lint` script
- [x] Self-review completed using the [steps outlined here](https://github.com/mongodb/node-mongodb-native/blob/HEAD/CONTRIBUTING.md#reviewer-guidelines)
- [x] PR title follows the [correct format](https://www.conventionalcommits.org/en/v1.0.0/): `type(NODE-xxxx)[!]: description`
  - Example: `feat(NODE-1234)!: rewriting everything in coffeescript`
- [x] Changes are covered by tests
- [x] New TODOs have a related JIRA ticket

### CODE REVIEW

File: test/integration/node-specific/examples/project_fields_from_query_results.test.js [hunk 1]
```diff
@@ -219,4 +219,45 @@ describe('examples(project-fields-from-query):', function () {
...
+            }
+          },
+          area: {
+            $concat: [{ $toString: { $multiply: ['$size.h', '$size.w'] } }, ' ', '$size.uom']
+          },
+          reportNumber: { $literal: 1 }
+        });
+      //  End Aggregation Projection Example 1
+      const docs = await cursor.toArray();
+      docs.forEach(function (doc) {
```
Comment (nbbeeken) [inline test/integration/node-specific/examples/project_fields_from_query_results.test.js:255]:
```suggestion
      for (const doc of docs) {
```

File: test/integration/node-specific/examples/project_fields_from_query_results.test.js [hunk 2]
```diff
@@ -219,4 +219,45 @@ describe('examples(project-fields-from-query):', function () {
       });
     }
   });
+
+  it('Aggregation Projection Example 1', {
+    metadata: { requires: { topology: ['single'], mongodb: '>= 4.4.0' } },
```
Comment (nbbeeken) [inline test/integration/node-specific/examples/project_fields_from_query_results.test.js:224]: Any reason to not run this on more topologies?

**Review by nbbeeken — CHANGES_REQUESTED** [review]

File: test/integration/node-specific/examples/project_fields_from_query_results.test.js (hunk 2 above)
Comment (aditi-khare-mongoDB) [reply test/integration/node-specific/examples/project_fields_from_query_results.test.js:224]: All other tests on this tests file also use a single topology, and they all live on the same docs page. I'm running evergreen patch to see which topologies support this functionality; and I'll get back to you once that's finished running.

Here's the [EVG Link](https://spruce.mongodb.com/version/65f496e252556600074d1da0/tasks?sorts=STATUS%3AASC%3BBASE_STATUS%3ADESC) (for reference).

File: test/integration/node-specific/examples/project_fields_from_query_results.test.js (hunk 1 above)
Comment (aditi-khare-mongoDB) [reply test/integration/node-specific/examples/project_fields_from_query_results.test.js]: Made the change!

**Review by nbbeeken — APPROVED** [review]

**Pull request merged**